packages/calcite-components/src/components/input-time-picker/input-time-picker.stories.ts

```diff
@@ -20,110 +20,44 @@ export const simple = (): string => html`
-    scale="${select("scale", ["s", "m", "l"], "m")}"
-    step="${number("step", 0.1)}"
-    value="${text("value", "10:37:09.5")}"
-  >
-  </calcite-input-time-picker>
+  <calcite-input-time-picker step="0.1" value="10:37:09.5" open> </calcite-input-time-picker>
 `;
 
 export const centiseconds_TestOnly = (): string => html`
-  <calcite-input-time-picker
-    ${boolean("disabled", false)}
-    ${boolean("hidden", false)}
-    name="${text("name", "simple")}"
-    ${boolean("open", true)}
-    placement="${select("placement", menuPlacements, defaultMenuPlacement)}"
-    scale="${select("scale", ["s", "m", "l"], "m")}"
-    step="${number("step", 0.01)}"
-    value="${text("value", "10:37:09.06")}"
-  >
-  </calcite-input-time-picker>
+  <calcite-input-time-picker step="0.01" value="10:37:09.06" open> </calcite-input-time-picker>
 `;
 
 export const milliseconds_TestOnly = (): string => html`
-  <calcite-input-time-picker
-    ${boolean("disabled", false)}
-    ${boolean("hidden", false)}
-    name="${text("name", "simple")}"
-    ${boolean("open", true)}
-    placement="${select("placement", menuPlacements, defaultMenuPlacement)}"
-    scale="${select("scale", ["s", "m", "l"], "m")}"
-    step="${number("step", 0.001)}"
-    value="${text("value", "10:37:09.023")}"
-  >
-  </calcite-input-time-picker>
+  <calcite-input-time-picker step="0.001" value="10:37:09.023" open> </calcite-input-time-picker>
 `;
 
 export const disabled_TestOnly = (): string =>
   html`<calcite-input-time-picker disabled scale="l" icon step="1" value="01:02"></calcite-input-time-picker>`;
 
 export const darkModeRTL_TestOnly = (): string => html`
-  <calcite-input-time-picker
-    ${boolean("disabled", false)}
-    ${boolean("hidden", false)}
-    class="calcite-mode-dark"
-    name="${text("name", "dark")}"
-    scale="${select("scale", ["s", "m", "l"], "m")}"
-    step="${number("step", 1)}"
-    value="${text("value", "22:37")}"
-  >
-  </calcite-input-time-picker>
+  <calcite-input-time-picker class="calcite-mode-dark" value="22:37" step="1"> </calcite-input-time-picker>
 `;
 
 darkModeRTL_TestOnly.parameters = { modes: modesDarkDefault };
 
 export const open_TestOnly = (): string => html`
-  <calcite-input-time-picker
-    name="${text("name", "placement-top")}"
-    value="${text("value", "10:37")}"
-    ${boolean("open", true)}
-  >
-  </calcite-input-time-picker>
+  <calcite-input-time-picker value="10:37" step="1" open> </calcite-input-time-picker>
 `;
 
-export const koreanLocale_TestOnly = (): string => html`
-  <calcite-input-time-picker
-    id="reference-element"
-    ${boolean("disabled", false)}
-    ${boolean("hidden", false)}
-    name="${text("name", "light")}"
-    scale="${select("scale", ["s", "m", "l"], "m")}"
-    step="${number("step", 1)}"
-    value="${text("value", "10:37")}"
-    lang="ko"
-    open
-  >
-  </calcite-input-time-picker>
+export const koreanLocaleInvalidStatus_TestOnly = (): string => html`
```

Comment:
Should the invalid status be a separate story to keep this one strictly focused on `ko`?

Reply:
Let's add a separate one. 

I agree we need to decrease snapshot count. Can you create an issue to do another pass to identify and consolidate stories? I think we could do so for anything testing different locales and scales.

Reply:
Sure. We can also combine multiple components with different states into a single story. Like a "locale" story with 5 different inputs of various locales and a "scale" story with all three scales.